app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -91,6 +91,9 @@ def timeout
         end
 
         def include_liveness?
```

Comment:
Same question as above: does this method add value? And if they both add value, let's call them the same thing.

Reply:
@soniaconnolly  - i see what you mean ... upon your suggestion, i attempted changing it but  soon realized that it will be too similar naming (_liveness_checking_required_ vs _liveness_checking_required?_) to the existing attribute name and open the opportunity  to a bad typo mistake where we evaluate whether or not to put a selfie in the payload